README.md

```diff
@@ -76,52 +76,57 @@ python -m pip install pennylane
 
 ## Docker support
 
-**Docker** support exists for building using **CPU** and **GPU** (Nvidia CUDA
-11.1+) images. [See a more detailed description
-here](https://pennylane.readthedocs.io/en/stable/development/guide/installation.html#docker).
+Docker images are found on the [PennyLane Docker Hub page](https://hub.docker.com/u/pennylaneai), where there is also a detailed description about PennyLane Docker support. [See description here](https://docs.pennylane.ai/projects/lightning/en/stable/dev/docker.html) for more information.
 
 ## Getting started
 
-For an introduction to quantum machine learning, guides and resources are available on
-PennyLane's [quantum machine learning hub](https://pennylane.ai/qml/):
+Get up and running quickly with PennyLane by following our [quickstart guide](https://docs.pennylane.ai/en/stable/introduction/pennylane.html), designed to introduce key features and help you start building quantum circuits right away.
+
+Whether you're exploring quantum machine learning (QML), quantum computing, or quantum chemistry, PennyLane offers a wide range of tools and resources to support your research:
 
 <img src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/master/doc/_static/readme/gpu_to_qpu.png" align="right" width="400px">

```

Comment:
@justinpickering is there a plan to update this graphic?

Reply:
@josh146 - @tarik-elkhateeb suggested this image (with transparent background) :) 

![image](https://github.com/user-attachments/assets/58bfd0f8-c1c9-455f-a365-10bd75945f7e)